compiler/record-hessian/include/record-hessian/HessianObserver.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __RECORD_HESSIAN_HESSIANOBSERVER_H__
+#define __RECORD_HESSIAN_HESSIANOBSERVER_H__
+
+#include "record-hessian/HessianComputer.h"
+
+#include <luci_interpreter/Interpreter.h>
+#include <luci_interpreter/core/Tensor.h>
```

Comment:
`core/Tensor.` doesn't seem to be used in this header

Reply:
or is `luci_interpreter/Interpreter.h` not used?

Reply:
I see `HessianComputer.h` includes luci-interpreter types but it would be better to include what is used in this header.